src/mumble/Settings.cpp

```diff
@@ -323,7 +323,9 @@ Settings::Settings() {
 	bOnlyAttenuateSameOutput       = false;
 	bAttenuateLoopbacks            = false;
 	iMinLoudness                   = 1000;
-	iVoiceHold                     = 50;
+	/// Actual mic hold time is (iVoiceHold / 100) seconds, where iVoiceHold is specified in 'frames',
+	/// each of which has a size of iFrameSize
```

Comment:
```suggestion
	/// each of which has a size of iFrameSize (see AudioInput.h)
```
I forgot that this variable is defined in a different header ^^